ocean_grid_generator.py

```diff
@@ -1108,24 +1118,17 @@ def main(
         # The phi resolution in the first and last row of Mercator grid along the symmetry meridian
         DeltaPhiMerc_so = phiMerc[1, Ni // 4] - phiMerc[0, Ni // 4]
         DeltaPhiMerc_no = phiMerc[-1, Ni // 4] - phiMerc[-2, Ni // 4]
-        # Start lattitude from dy above the last Mercator grid
-        lat0_bp = phiMerc[-1, Ni // 4] + DeltaPhiMerc_no
-        if refineR == 1 and enhanced_equatorial:
-            lat0_bp = phiMerc[-1, Ni // 4]  # SPEAR
+        # The bopolar grid should start from the same lattitude that Mercator ends.
+        # Then when we stitch the two subgrids we should drop the x,y,dx,angle from one of the two.
+        # This way we get a continous dy and area.
+        lat0_bp = phiMerc[-1, Ni // 4] 
```

Comment:
This is the main fix (deleting  + DeltaPhiMerc_no) , the rest is cosmetic changes.